dockers/docker-base-bullseye/sources.list.armhf

```diff
@@ -4,6 +4,7 @@
 deb [arch=armhf] http://deb.debian.org/debian bullseye main contrib non-free
 deb [arch=armhf] http://deb.debian.org/debian bullseye-updates main contrib non-free
 deb [arch=armhf] http://security.debian.org bullseye-security main contrib non-free
+deb [arch=armhf] http://deb.debian.org/debian bullseye-updates main contrib non-free
```

Comment:
Line duplication (bullseye-updates).

Reply:
@kv-y , it is different mirror with the same name bullseye-update, it is not a duplicate. Could you please double check?

Reply:
@xumia 
line 5: deb [arch=armhf] http://deb.debian.org/debian bullseye-updates main contrib non-free
line 7: deb [arch=armhf] http://deb.debian.org/debian bullseye-updates main contrib non-free

Reply:
@kv-y , fixed, thanks.